### PR TITLE
Added 'allowed countries' to the reference

### PR DESCRIPTION
**Pull request by dheesen**
## Purpose of this pull request

I added the 'allowed countries' setting to the reference for the Scope Quick Reference. As we discussed a Magento website structure earlier today, this was easily overseen in the discussion around this subject. It's an important setting to be aware of when you're talking about internationalization.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/stores/store-scope-reference.html

whatsnew
Added the _Allowed countries_ setting to the website scope section in the [Scope Quick Reference](https://docs.magento.com/user-guide/stores/store-scope-reference.html).

### CODE REVIEW

**Comment by leslietilling** [comment]
@dheesen  Please sign the Adobe CLA so that we can process your PR: http://opensource.adobe.com/cla.html

**Comment by dobooth** [comment]
Hi @dheesen. Thanks for the contribution. Please sign the Adobe CLA so we can process this PR.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

File: src/stores/store-scope-reference.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ group: getting-started
 |Product Prices|[Product prices]({% link catalog/catalog-price-scope.md %}) can be configured to be applied at either a global or website level.|
 |Payment Methods|[Payment methods]({% link payment/payments.md %}) are configured at the website level, although the title and instructions can be configured for each store view.|
 |Checkout|The [checkout process]({% link sales/checkout-process.md %}) takes place at the website level, although some display options can be configured for each store view. All stores associated with a website have the same [checkout configuration]({% link sales/checkout-options.md %}).|
+|Allowed countries|Allowed countries can be configured on the website level. The [allowed countries]({% link stores/country-options.md}) settings are used in the checkout to limit where a customer can come from.|
```
Comment (leslietilling) [inline src/stores/store-scope-reference.md:23]: fix syntax

```suggestion
|Allowed countries|Allowed countries can be configured on the website level. The [allowed countries]({% link stores/country-options.md %}) settings are used in the checkout to limit where a customer can come from.|
```

**Comment by leslietilling** [comment]
running tests

**Review by leslietilling — APPROVED** [review]

**Pull request merged**